.github/workflows/ci.yml

```
@@ -3,13 +3,21 @@ on:
+
 jobs:
   prepare:
     runs-on: ubuntu-latest
     outputs:
       ui: ${{ steps.ui.outputs.any_changed }}
       api: ${{ steps.api.outputs.any_changed }}
       subscriber: ${{ steps.subscriber.outputs.any_changed }}
+      policies: ${{ steps.policies.outputs.any_changed }}
       engine: ${{ steps.engine.outputs.any_changed }}
       websocket: ${{ steps.websocket.outputs.any_changed }}
       policies: ${{ steps.policies.outputs.any_changed }}
```

actionlint (1.7.4)

key "policies" is duplicated in "outputs" section. previously defined at line:20,col:7. note that this key is case insensitive

(syntax-check)